svelte-kit-scss/src/lib/components/Profile/UserProfile/UserProfile.spec.ts

```diff
@@ -20,15 +20,20 @@ describe('UserProfile', () => {
     ['name', mockUserProfile.name],
     ['login', mockUserProfile.login],
     ['bio', mockUserProfile.bio],
-    // ['followers', mockUserProfile.followers],
-    // ['following', mockUserProfile.following],
-    // ['email', mockUserProfile.email],
-    // ['blog', mockUserProfile.blog],
-    // ['twitterUsername', mockUserProfile.twitterUsername],
-    // ['location', mockUserProfile.location],
+    ['followers', mockUserProfile.followers],
+    ['following', mockUserProfile.following],
+    ['email', mockUserProfile.email],
+    ['blog', mockUserProfile.blog],
+    ['twitterUsername', mockUserProfile.twitterUsername],
+    ['location', mockUserProfile.location],
   ])('should render: %s', (testId, expectedValue) => {
     const element = screen.getByTestId(testId);
     const expectedText = String(expectedValue);
```

Comment:
Why do we manually "cast" this to string? Is there behaviour that would have this not being a string?